### PR TITLE
Japanese ROM diff

### PR DESCRIPTION
**Pull request by BlueSpeedsterYT**
This aims at making the game compile a Japanese ROM, this may not be accurate but help on this will be needed

### CODE REVIEW

File: src/game/time_attack/results.c [hunk 1]
```diff
@@ -288,27 +288,27 @@ void sub_80897E8(void)
...
         for (i = 0; i < 8; i++) {
-            s->x = temp + i * 0x20;
+            s->x = temp + (i * 32);
             DisplaySprite(s);
         }
     } else {
         s = &resultsCutScene->unk12C;
         for (i = 0; i < 8; i++) {
-            s->x = i << 5; // TODO: Does (i * 32) match?
+            s->x = -(i * 32); // TODO: Does (i * 32) match?
```
Comment (freshollie) [inline src/game/time_attack/results.c:306]: I think the `-` may be the reason for the non match?

File: .gitignore [hunk 2]
```diff
@@ -49,13 +49,14 @@ Temporary Items
 # =========================
 # Build Outputs
 # =========================
+
 *.gba
 *.sav
 *.elf
 *.map
 *.bin
+!data/**/*.bin
```
Comment (freshollie) [inline .gitignore:58]: Why this change?

Comment (BlueSpeedsterYT) [reply .gitignore:58]: no idea

Comment (freshollie) [reply .gitignore:58]: Well, change it back then plz

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -90,6 +90,11 @@ jobs:
           echo "REPORTS_COMMIT_MSG=$( git log --format=%s ${GITHUB_SHA} )" >> $GITHUB_ENV
           cp sa2.map gh-pages/maps/${GITHUB_SHA}.map
 
+      - name: Japan
+        run: |
+          make clean
+          make japan -j${nproc}
+
```
Comment (freshollie) [inline .github/workflows/build.yml:97]: Can you comment this out for now, as it's going to fail once merged to main

File: .gitignore [hunk 4]
```diff
@@ -72,5 +72,4 @@ reports
 ./maps
 *.4bpp
 *.8bpp
-
-!data/maps/*/*/tilemaps/*/*.bin
```
Comment (freshollie) [inline .gitignore:76]: This needs to be kept I think

File: .gitignore [hunk 5]
```diff
@@ -49,13 +49,13 @@ Temporary Items
...
 # Build Outputs
 # =========================
+
 *.gba
 *.sav
 *.elf
 *.map
 *.bin
 
-*.a
```
Comment (freshollie) [inline .gitignore:58]: And this too

**Review by freshollie — APPROVED** [review]
